jsonrpc/server.go

```diff
@@ -438,7 +431,10 @@ func (s *Server) handleRequest(ctx context.Context, req *Request) (*response, er
 	if errAny := tuple[1].Interface(); !isNil(errAny) {
 		res.Error = errAny.(*Error)
 		if res.Error.Code == InternalError {
-			s.listener.OnRequestFailed(req.Method, err)
+			s.listener.OnRequestFailed(req.Method, res.Error)
+			reqJSON, _ := json.Marshal(req)
+			errJSON, _ := json.Marshal(res.Error)
+			s.log.Debugw("Failed handing RPC request", "req", string(reqJSON), "res", string(errJSON))
```

Comment:
Might be good to promote this to a warning.

Reply:
I would rather stay with debug. If we label it as a warning, we might start overlooking real warnings, especially if these pop up a lot.